tests/search/attributebitvectors/attributebitvectors.sd

```diff
@@ -0,0 +1,32 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+schema attributebitvectors {
```

Comment:
Consider simplifying by renaming to test.sd, _schema test_, and _document test_.